src/modules/display.js

```diff
@@ -23,7 +22,7 @@ const display = async () => {
     tbody.append(tr);
   });
 };
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commits capitalized to make your pull request descriptive and professional 👍